mkdocs.yml

```diff
@@ -1,41 +1,40 @@
 site_name: ElectionGuard
 # site_description:
-site_author: Microsoft
+# site_author: Microsoft
 # google_analytics:
 # remote_branch: for gh-deploy to GithubPages
 # remote_name: for gh-deploy to Github Pages
-copyright: "© Microsoft 2021"
+# copyright: "© Microsoft 2021"
 docs_dir: "docs"
 repo_url: https://github.com/microsoft/electionguard/
 nav:
   - About: 
     - Overview: index.md
     - Roadmap: Roadmap.md
-  - Guide:
-    - Overview: guide/Overview.md
-    - SDK: 
-      - Core Component: guide/Core_Component.md
-      - Implementations: guide/Implementations.md
-    - Additional Topics:
-      - Verifiable Election: guide/Verifiable_Election.md
-      - Election Manifest: guide/Election_Manifest.md
-      - Precinct Scan Use Case: guide/Precinct_scan.md
-      - Glossary: guide/Glossary.md 
-  - Specification:
-    - Overview: spec/Overview.md
-    - v0.95:
-      - Overview: spec/0.95.0/1_Overview.md
-      - Structure: spec/0.95.0/2_Structure.md
-      - Baseline Parameters: spec/0.95.0/3_Baseline_parameters.md     
-      - Key Generation: spec/0.95.0/4_Key_generation.md     
-      - Ballot Encryption: spec/0.95.0/5_Ballot_encryption.md          
-      - Ballot Aggregation: spec/0.95.0/6_Ballot_aggregation.md     
-      - Verifiable Decryption: spec/0.95.0/7_Verifiable_decryption.md     
-      - The Election Record: spec/0.95.0/8_The_election_record.md     
-      - Verifier Construction: spec/0.95.0/9_Verifier_construction.md 
-      - Applicability to E2E Verifiability and RLA: spec/0.95.0/10_Applicability.md
-      - Acknowledgements: spec/0.95.0/11_Acknowledgements.md
+    - Contributions: Contributions.md
```

Comment:
I believe this should be mentioned on the overview. For example on the `Overview: index.md` as a section then this should link to another page. 

1. These need to be mentioned in the Developer Guide first and foremost. 
2. We should make a Community tab and just include these there.